client/txn.go

```diff
@@ -295,6 +295,10 @@ func (txn *Txn) ReverseScan(begin, end interface{}, maxRows int64) ([]KeyValue,
 	return txn.scan(begin, end, maxRows, true)
 }
 
+// TODO(pmattis): Txn.ReverseScan is neither used or tested. Silence unused
+// warning.
+var _ = (*Txn)(nil).ReverseScan
```

Comment:
You can't just slap it into some test? Weird that we don't use it. Wouldn't SQL need it for index scans?